moto/rds/models.py

```diff
@@ -2886,6 +2887,48 @@ def create_db_parameter_group(
         self.db_parameter_groups[db_parameter_group_id] = db_parameter_group
         return db_parameter_group
 
+    def copy_db_parameter_group(
+        self,
+        source_db_parameter_group_identifier: str,
+        target_db_parameter_group_identifier: str,
+        target_db_parameter_group_description: str,
+        tags: Optional[List[Dict[str, str]]] = None,
+    ) -> DBParameterGroup:
+        if source_db_parameter_group_identifier.startswith("arn:aws:rds:"):
+            source_db_parameter_group_identifier = (
+                source_db_parameter_group_identifier.split(":")[-1]
+            )
+        if source_db_parameter_group_identifier not in self.db_parameter_groups:
+            raise DBParameterGroupNotFoundError(source_db_parameter_group_identifier)
+        if target_db_parameter_group_identifier in self.db_parameter_groups:
+            raise DBParameterGroupAlreadyExistsError(
+                target_db_parameter_group_identifier
+            )
+        source_db_parameter_group = self.db_parameter_groups[
+            source_db_parameter_group_identifier
+        ]
+
+        target_db_parameter_group = DBParameterGroup(
+            backend=source_db_parameter_group.backend,
+            db_parameter_group_name=target_db_parameter_group_identifier,
+            db_parameter_group_family=source_db_parameter_group.family,
+            description=target_db_parameter_group_description,
+            tags=tags,
```

Comment:
What's the AWS behavior with tags here?  If the source parameter group has tags will those be copied over to the target, or does the target only get tags if passed in?  Whatever the behavior, can you add test coverage to make it explicit?